src/HistoryEnter/page/index.tsx

```diff
@@ -8,9 +13,18 @@ function HistoryEnter() {
     <React.Fragment>
       <Header headerTitle="내 기록 보기" />
       <S.HistoryEnterPageBodyWrapper>
-        <S.Tab>즐겨찾기 한 레큐북</S.Tab>
-        <S.Tab>내가 만든 레큐북</S.Tab>
-        <S.Tab>내가 남긴 레큐노트</S.Tab>
+        <S.Tab variant="book">
+          <S.Text>즐겨찾기한 레큐북</S.Text>
+          <ImgMypageFavoriteLecueBook />
+        </S.Tab>
+        <S.Tab variant="book">
+          <S.Text>내가 만든 레큐북</S.Text>
+          <ImgMypageMakeLecueBook />
+        </S.Tab>
+        <S.Tab variant="letter">
+          <S.Text>내가 남긴 레터</S.Text>
+          <ImgMypageLetter />
+        </S.Tab>
```

Comment:
P5: 여기를 하나의 객체 배열로 묶어서 정의해준 뒤, map을 활용해서 렌더링해주면 중복코드를 제거할 수 있을 것 같아요! 물론 지금은 렌더링 하는 요소가 많지 않고 앞으로 데이터가 크게 변할 일도 없긴 하겠지만..! 상수 파일에 분리해서 정의해준다면 데이터 유지보수에도 더 좋을 것 같습니다! 😊
(아래 코드는 vscode로 돌려보고 적은게 아니라서 단순히 참고 정도만 해주시면 좋을 것 같습니당 !)

```suggestion
const DATA = [
          { title: "즐겨찾기한 레큐북",
            variant: "book",
            img: <ImgMypageFavoriteLecueBook /> },
          { title: "내가 만든 레큐북",
            variant: "book",
            img: <ImgMypageMakeLecueBook /> },
          { title: "내가 남긴 레터",
            variant: "letter",
            img: <ImgMypageLetter /> } 
        ];
        
        DATA.map((data) => {
          return (
            <S.Tab variant={data.variant}>
               <S.Text>{data.title}</S.Text>
               {data.img}
             </S.Tab>
           )
         };
```

Reply:
헉 이렇게 반복되는 패턴이 있으면 이렇게 하면 .. 정말 유지보수 관점에서 좋을거 같네요 ..!! 수정해보도록 하겠습니다❤️